charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -9,6 +9,10 @@ clusterName: ""
 # Extra environment variables to add to each collector, bridge and instrumentation
 extraEnvs: []
 
+# Enables a cleanup job to make sure the CRs are uninstalled before the operator
+cleanupJob:
+  enabled: true
```

Comment:
Let's add a comment that enabling this is necessary for `helm uninstall` to work.  Users should really never be disabling this.